packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service-v2.ts

```diff
@@ -4,6 +4,7 @@ import { AwsLogDriver, Cluster, ContainerImage, FargateTaskDefinition, Propagate
 import { CompositePrincipal, Role, ServicePrincipal } from '@aws-cdk/aws-iam';
 import { Duration, Stack } from '@aws-cdk/core';
 import { Test } from 'nodeunit';
+import {FargatePlatformVersion} from '../../../aws-ecs/lib/fargate/fargate-service';
 import { ApplicationMultipleTargetGroupsFargateService, NetworkMultipleTargetGroupsFargateService } from '../../lib';
```

Comment:
You already have the `@aws-cdk/aws-ecs` module imported, you should add it above 

update 
`import { AwsLogDriver, Cluster, ContainerImage, FargateTaskDefinition, PropagatedTagSource, Protocol } from '@aws-cdk/aws-ecs';` 
to 
`import { AwsLogDriver, Cluster, ContainerImage, FargatePlatformVersion, FargateTaskDefinition, PropagatedTagSource, Protocol } from '@aws-cdk/aws-ecs';`